package.json

```diff
@@ -56,6 +57,14 @@
     "postcss": "^7.0.35",
     "prettier": "2.2.1",
     "react-query": "^3.16.0",
-    "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.1.2"
+    "styled-components": "^5.3.0",
+    "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.1.2",
```

Comment:
you should just be able to use the normal tailwindcss now since twin.macro will be compiling it all